applications/mne_scan/plugins/brainflowboard/brainflowboard.pro

```diff
@@ -0,0 +1,76 @@
+include(../../../../mne-cpp.pri)
+
+QT += core widgets
+
+TEMPLATE = lib
+DEFINES += BRAINFLOWBOARD_LIBRARY
+
+CONFIG += c++11
+
+DEFINES += QT_DEPRECATED_WARNINGS
+
+DESTDIR = $${MNE_BINARY_DIR}/mne_scan_plugins
+
+contains(MNECPP_CONFIG, static) {
```

Comment:
line 14 to 19 can be removed. plugins do not need to be build statically. this was a mistake introduced by me some time ago.

Reply:
fixed